Runtime/Code/Luau/LuauCoreCallbacks.cs

```diff
@@ -1157,10 +1157,8 @@ private static void SetComponentEnabledCallback(IntPtr thread, int instanceId, i
     static unsafe int CallMethodCallback(LuauContext context, IntPtr thread, int instanceId, IntPtr classNamePtr, int classNameSize, IntPtr methodNamePtr, int methodNameLength, int numParameters, IntPtr firstParameterType, IntPtr firstParameterData, IntPtr firstParameterSize, IntPtr firstParameterIsTable, IntPtr shouldYield) {
         CurrentContext = context;
         
-        // if (s_shutdown) return 0;
         Marshal.WriteInt32(shouldYield, 0);
         if (!IsReady) {
-            Profiler.EndSample();

```

Comment:
Removing this since it seems to be an artifact of the past. No need for it, and there's no matching BeginSample for it.